src/main/java/com/iota/iri/crypto/ISS.java

```diff
@@ -142,10 +237,31 @@
         return signatureFragment;
     }
 
-    public static byte[] digest(SpongeFactory.Mode mode, final byte[] normalizedBundleFragment, final byte[] signatureFragment) {
+    // Validation
```

Comment:
Is this supposed to start a code block?

`\\*****************************************Validation******************************************\\`
Doing `\\` means it won't be a javadoc

Reply:
// :p